proto/go_proto_library.bzl

```diff
@@ -213,6 +235,26 @@ def go_proto_library(name, srcs = None, deps = None,
       **kwargs
   )
 
+def _well_known_import_key(name):
+  return "%s%s:go_default_library" % (_well_known_repo, name)
+
+_well_known_imports = ["any", "duration", "empty", "struct", "timestamp", "wrappers"]
+
+# If you have well_known proto deps, rules_go will add a magic
+# google/protobuf/ directory at the import root
+def go_google_protobuf(name = _GO_GOOGLE_PROTOBUF):
+  deps = [_add_target_suffix(_well_known_import_key(wk), _PROTOS_SUFFIX)
+          for wk in _well_known_imports]
+  outs = [wk + ".proto" for wk in _well_known_imports]
+
+  native.genrule(
+      name = name,
+      srcs = deps,
+      outs = ["google/protobuf/"+o for o in outs],
+      cmd = "cp $(SRCS) $(@D)/google/protobuf/",
```

Comment:
No, because in  google/protobuf these files are in external/com_github_google_protobuf/src/google/protobuf